include/aws/iot/MqttClient.h

```diff
@@ -120,6 +120,15 @@ namespace Aws
                 Crt::Io::ClientBootstrap *bootstrap,
                 Crt::Allocator *allocator = Crt::g_allocator) noexcept;
 
+            /**
+             * Create a websocket configuration for use with the default credentials provider chain and default
+             * ClientBootstrap. Signing region will be used for Sigv4 signature calculations.
```

Comment:
Try and get doxygen to do a link to the original `ApiHandle::GetOrCreateDefaultXYZ()` function. Where we should have more documentation about "default XYZ"

you can update the docs by installing Doxygen and running
```
cd docsrc
doxygen Doxyfile
```

then you can look at the webpages that appear under the `docs/` folder locally to see if they look good